pkg/kubectl/cmd/create.go

```diff
@@ -167,6 +168,23 @@ See http://releases.k8s.io/HEAD/docs/user-guide/services-firewalls.md for more d
 				makePortsString(obj.Spec.Ports, true))
 			out.Write([]byte(msg))
 		}
+
+		_, ok := obj.Annotations[service.AnnotationLoadBalancerSourceRangesKey]
+		if ok && len(obj.Spec.LoadBalancerSourceRanges) > 0 {
+			msg := fmt.Sprintf(`You have specified both LoadBalancerSourceRanges field and annotation [service.beta.kubernetes.io/load-balancer-source-ranges].
```

Comment:
if we're going to do this sort of message, why not just print for the annotation, regardless of whether they have both?  You're using a deprecated annotation, please use the field instead.